package.json

```diff
@@ -2,18 +2,17 @@
   "name": "@kontist/mock-solaris",
   "version": "1.0.11",
   "description": "Mock Service for Solaris API",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
   "repository": "https://github.com/kontist/mock-solaris.git",
   "author": "Kontist GmbH",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "tsc && cp -r src/templates dist",
-    "start": "node dist/server.js",
-    "debug": "node --nolazy --inspect=0.0.0.0:9559 dist/server.js",
-    "test": "npm run lint && echo \"Error: no test specified\" && exit 1",
-    "transpile": "tsc && echo \"--ALL FILES TRANSPILED!\"",
-    "transpile-watch": "tsc --watch",
+    "build": "tsc && cp -r src/templates dist/src",
+    "start": "node dist/src/server.js",
+    "debug": "node --nolazy --inspect=0.0.0.0:9559 dist/src/server.js",
+    "test": "npm run lint && npm run build && mocha --recursive 'dist/tests/**/*.spec.js'",
+    "cover": "nyc npm run test",
```

Comment:
what do you think of changing it to:

`"coverage": "nyc --check-coverage=false npm run test"`?

This way we can always run `coverage` (including the CI) to get the latest report, plus we won't have to deal with the non-zero status code that `nyc` returns when coverage is below a certain threshold.

Reply:
"cover" -> "coverage" 👍
But I actually like the non-zero status code, that way new code should not lower the coverage. Probably it only makes sense if the coverage will be a bit higher. Disabled it for now, as you suggested, see https://github.com/kontist/mock-solaris/pull/5/commits/99d694edd9b15ea3aba7bf989e8eb4eb0e4ef602

Reply:
Thanks! I totally agree with coverage breaking a certain build pipeline, but I think that should be enabled once we reach a certain coverage percentage and want to maintain it. For now, at 6.x%, it doesn't really make much sense.